pkg/proxy/proxy_test.go

```diff
@@ -33,7 +37,7 @@ var _ = Describe("Setup", func() {
 
 		uo := unstructured.Unstructured{Object: m}
 
-		err = proxy.Setup(&uo)
+		err = proxyStruct.Setup(&uo)
```

Comment:
Can we make `Setup()` a private method if it is not part of the API?

Reply:
it is part of the API, it is defined in the interface

Reply:
Sorry, I missed that.